.travis.yml

```diff
@@ -3,24 +3,20 @@ sudo: false
 python: 2.7
 env:
     - TOX_ENV=flake8
-    - TOX_ENV=py27-django16
     - TOX_ENV=py27-django17
     - TOX_ENV=py27-django18
     - TOX_ENV=py27-djangomaster
-    - TOX_ENV=py35-django16
-    - TOX_ENV=py35-django17
-    - TOX_ENV=py35-django18
-    - TOX_ENV=py35-djangomaster
+    - TOX_ENV=py34-django17
+    - TOX_ENV=py34-django18
+    - TOX_ENV=py34-djangomaster
 matrix:
     allow_failures:
         - env: TOX_ENV=flake8
         - env: TOX_ENV=py27-django17
```

Comment:
Not sure. It may be passing now, but since we don't actually _need_ it to pass (as we are targeting 1.8) shouldn't it remain there in case a future update breaks the 1.7 compatibility?